proxy/nasshp/browser.go

```diff
@@ -22,15 +22,16 @@ type ReplaceableBrowser struct {
 	log logger.Logger
 
 	notifier chan error
-	wc       *websocket.Conn
-	err      error
+	wc       *websocket.Conn  // Protected by lock.
+	err      error            // Protected by lock.
```

Comment:
nit: Group these with the mutex to denote what vars are protected by which mutex, if any

Reply:
done.